client/style/calendar.css

```diff
@@ -0,0 +1,128 @@
+
+.reservation-popup-date-picker {
+  /* position: relative; */
+  width: 100%;
+  /* max-width: 33.33333%; */
+  /* width: 333px; */
+  margin: 0 auto;
+}
+
+.goog-date-picker {
+  position: absolute;
+  background: #fff;
+  border: 1px solid #ccc;
+  border-radius: 6px;
+  -webkit-box-shadow: 0 1px 2px rgba(10,10,10,0.2);
+  box-shadow: 0 1px 2px rgba(10,10,10,0.2);
+  text-align: center;
+  vertical-align: middle;
+  cursor: default;
+  z-index: 1054;
+  outline: none;
+  top: 35px;
+}
+
+.reservation-popup-date-picker .goog-date-picker-head td {
+  background: white;
+}
+
+.reservation-popup-date-picker .goog-date-picker-monthyear {
+  font-weight: bold;
+  padding: 15px;
+}
+
+.goog-date-picker td {
+  padding: 3px;
+}
+
+th, td {
+  vertical-align: middle;
+}
```

Comment:
Should avoid using plain html element selectors. They will break all of our  pages when we combine for the proxy service.